.github/workflows/stale.yml

```diff
@@ -13,5 +13,7 @@ jobs:
         stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. It will be closed in 14 days if no further activity occurs. Thank you for your contribution.'
         days-before-stale: 30
         days-before-close: 14
-        only-labels: 'question'
-
+        only-issue-labels: 'question'
+        stale-pr-message: 'This Pull Request has been automatically marked as stale because it has not had recent activity. It will be closed in 21 days if no further activity occurs. Remember, a closed PR can always be reopened. Thank you for your contribution.'
+        days-before-pr-stale: 60
+        days-before-pr-close: 21
```

Comment:
How about adding a message while closing the pull request using [_close-pr-message_](https://github.com/actions/stale#close-pr-message)?

Reply:
Added.